pkg/envoy/errors.go

```diff
@@ -14,4 +14,13 @@ var (
 
 	// ErrIncorrectlyInitializedConfigGetter is an error
 	ErrIncorrectlyInitializedConfigGetter = errors.New("incorrectly initialized config getter")
+
+	// ErrNoDestinationEndpoints is an error
+	ErrNoDestinationEndpoints = errors.New("no destination endpoints")
+
+	// ErrUnmarshalingClusterLoadAssigment is an error
+	ErrUnmarshalingClusterLoadAssigment = errors.New("error unmarshaling envoy cluster load assigment")
+
+	// ErrEndpointNotFound is an error
+	ErrEndpointNotFound = errors.New("endpoint not found")
 )
```

Comment:
Can the comments for errors.go be more detailed? Or if it's redundant can we not do the comments?

Reply:
Will augment these!